README.md

```diff
@@ -24,7 +24,9 @@ source code repository.
 Run master and workers against your local Olympia docker container (by IP
 address, which may be different for you).
 
-    SITE_UNDER_TEST=http://192.168.59.103 docker-compose up -d
+    SITE_UNDER_TEST=http://$(docker-machine ip default) \
```

Comment:
this only works for `docker-machine` though :) I guess that's the recommended approach so that's fine.